src/Compiler/CompilerOptions.cpp

```diff
@@ -76,6 +76,11 @@ llvm::cl::opt<std::string> shapeInformation("shapeInformation",
         "unknown dimensions)"),
     llvm::cl::value_desc("value"), llvm::cl::cat(OnnxMlirOptions));
 
+llvm::cl::opt<std::string> menvVarName("menvVarName",
```

Comment:
the `-m` was principally use for "machine", namely `-march` `-mtripple`, `-mcpu`...

Since the original env var is named ONNX_MLIR_FLAGS, maybe `-custom-env-flags`?

Reply:
I'm sorry I didn't notice that just now. The new PR is #1588